lib/common/util.sh

```diff
@@ -314,3 +314,20 @@ unzip_file() {
     )
 }
 export -f unzip_file
+
+# delete files in directory that start like the given file name
+# (useful to purge error directory when same file is eventually
+# successfully processed)
+# $1 - directory
+# $2 - file name (basename)
+delete_files_in_dir() {
```

Comment:
Would it make sense to restrict this to just deleting files in an error directory (e.g. `delete_from_error_dir`)? I.e. the first arg could just be the pipeline job name (ACORN in this case) which should be a sub-directory of `$ERROR_DIR`. Might just be a bit safer.

Actually you might not even need the first arg because there's an env variable called `$JOB_NAME`.